hdtest/source/embeddedtest.d

```diff
@@ -74,6 +75,14 @@ class EmbeddedTest : HibernateTest {
         Customer c2 = r2.uniqueResult!Customer();
         assert(c2 !is null);
         assert(c2.billingAddress.street == "101001 Robotface");
+
+        // Make sure queries on strings with (') characters work.
+        auto r3 = sess.createQuery("FROM Customer WHERE name = :Name")
+                .setParameter("Name", "Erdrick O'Henry");
+        Customer c3 = r3.uniqueResult!Customer();
+        import std.stdio;
+        writeln("c3.name = ", c3.name);
```

Comment:
rather than a writeln perhaps best having an assert that the c2 name and c3 name match

Reply:
Oops, that was leftover debugging code, I'll just take that out.